tests/my_house_tests.py

```diff
@@ -148,3 +166,36 @@ def test_delete_expenses_of_my_house(self, client, house_factory, expense_factor
         assert len(Expenses.objects.all()) == 0
         assert response.status_code == 302
         assert response.url == '/../house'
+
+    def test_edit_existing_house(self, client, house_factory, new_user):
+        client.force_login(new_user)
+        country = Country.create_country(COUNTRY)
+        house = House.create_house(
```

Comment:
see here
it should go to fixture, 
return the House.create_house(

You are calling it more than once in this files

Reply:
I'm calling to this function only in line 173, 
Do I need to create the fixture anyway?
I called one time to create_house, one time to create_country and one time to create_city
@Yarboa

Reply:
As far i see
created twice in line 19 and 177
Instead of function call a fixture with the return value